lib/rules/selector-pseudo-class-allowed-list/__tests__/index.mjs

```diff
@@ -49,6 +51,16 @@ testRule({
 			code: 'html { --custom-property-set: {} }',
 			description: 'custom property set in selector',
 		},
+		{
+			code: stripIndent`
+				/* stylelint-disable-next-line selector-pseudo-class-allowed-list */
+				:HOVER, /* stylelint-disable-next-line selector-pseudo-class-allowed-list */
+				:HOVER,
+				/* stylelint-disable-next-line selector-pseudo-class-allowed-list */
+				:HOVER
+				{}
+			`,
```

Comment:
Maybe better to use `:focus` in the tests.

That `:hover` and `:HOVER` are treated differently in this rule is peculiar and not what we want to test in this case is:)

Reply:
Ah, that sounds good. I've fixed it.

[chore: rename :HOVER to :focus](https://github.com/stylelint/stylelint/pull/8061/commits/7de08c98903b032bcfad92e5e1566b8423dab9c0)